pkg/Microsoft.Private.CoreFx.NETCoreApp/netcoreapp.rids.props

```diff
@@ -28,5 +28,8 @@
     <OfficialBuildRID Include="tizen.4.0.0-armel">
       <Platform>armel</Platform>
     </OfficialBuildRID>
+    <OfficialBuildRID Include="tizen.5.0.0-armel">
```

Comment:
Should we just replace the tizen.4.0.0-armel with tizen.5.0.0-armel?

Reply:
@weshaggard both RIDs are required

Reply:
Sounds good. That implies you still support both 4.0.0 and 5.0.0 with the current master builds. Are there plans to support the latest .NET Core on 5.0.0 only in the future?

Reply:
@weshaggard until the official release of Tizen 5.0 in autumn we have to support both. After that it is yet to be decided.